### ISSUE TITLE
Rename Service

### ISSUE THREAD

**Issue by Solido** [matanlurey/cynic#4]
Service is a too generic name and it's difficult to recognise in code base
Cynic or OnlineService would help readability.

Could provide PR if idea is validated.

```dart
static const servers = const [
    Cynic.googleDns, // insted of const Service.forIp('8.8.8.8', name:'google'),
    const Cynic.forIp('192.168.66.221', name:'api'),
    const Cynic.forIp('192.168.66.222', name:'auth'),
    const Cynic.forIp('192.168.66.223', name:'cold'),
  ];
```

**Comment by matanlurey** [comment]
Hmm, OK. I'm fine with renaming.

I'd hate to call it `Cynic` though, because that sort of makes adding other classes or utilities to this library seem second-class or out of place. How about either:

* `OnlineService`
* `ServiceStatus`
* `StatusChecker`

Looking at [npm for ideas](https://www.npmjs.com/search?q=check+online):

* `Reachable` stands out to me:

```dart
const Reachable.forIp('1.1.1.1')
```

How does that sound @Solido?

**Comment by Solido** [comment]
Reachable ! no doubt.

```dart
const Reachable.ip('1.1.1.1')
```
Concision and expressivity are first citizen.
What do you think of also shorten forIp as long there's no confusion ?

**Comment by matanlurey** [comment]
Like Reachable.ip? Go for it.

**Comment by matanlurey** [comment]
Closed by #6.
